iothub/service/src/ServiceClient.cs

```diff
@@ -272,7 +272,9 @@ public virtual async Task OpenAsync()
         {
             Logging.Enter(this, $"Opening AmqpServiceClient", nameof(OpenAsync));
 
-            await _faultTolerantSendingLink.OpenAsync(_openTimeout).ConfigureAwait(false);
+            using var ctx = new CancellationTokenSource(_openTimeout);
+
+            await _faultTolerantSendingLink.OpenAsync(ctx.Token).ConfigureAwait(false);
```

Comment:
This method no longer supports timespans.

Reply:
oh, breaking change for the amqp library then!

Reply:
:D